zerver/webhooks/netlify/fixtures/created.json

```diff
@@ -0,0 +1,52 @@
+{
+   "id":"5b78192ac9659217dbf7c9aa",
+   "site_id":"573f11b2-f5f3-481f-a1f8-85feb457ff49",
+   "build_id":"5b78192ac9659217dbf7c9ab",
+   "state":"ready",
+   "name":"objective-jepsen-35fbb2",
+   "url":"http://objective-jepsen-35fbb2.netlify.com",
+   "ssl_url":"https://objective-jepsen-35fbb2.netlify.com",
+   "admin_url":"https://app.netlify.com/sites/objective-jepsen-35fbb2",
+   "deploy_url":"http://5b78192ac9659217dbf7c9aa.objective-jepsen-35fbb2.netlify.com",
+   "deploy_ssl_url":"https://5b78192ac9659217dbf7c9aa--objective-jepsen-35fbb2.netlify.com",
+   "created_at":"2018-08-18T13:03:38.315Z",
+   "updated_at":"2018-08-18T13:03:44.609Z",
+   "user_id":"5b68ca02b13fb17905788d44",
+   "error_message":null,
+   "required":[
+
+   ],
+   "required_functions":[
+
+   ],
+   "commit_ref":null,
+   "review_id":null,
+   "branch":"master",
+   "commit_url":null,
+   "skipped":null,
+   "locked":null,
+   "log_access_attributes":{
+      "type":"firebase",
+      "url":"https://netlify.firebaseio.com/builds/5b78192ac9659217dbf7c9ab/log"
+   },
+   "title":null,
+   "review_url":null,
+   "published_at":"2018-08-18T13:03:42.462Z",
+   "context":"production",
+   "deploy_time":3,
+   "available_functions":[
+
+   ],
+   "summary":{
+      "status":"ready",
+      "messages":[
+         {
+            "type":"info",
+            "title":"No new files created for this deploy",
+            "description":null,
+            "image":null
+         }
+      ]
+   },
+   "screenshot_url":"https://353a23c500dde3b2ad58-c49fe7e7355d384845270f4a7a0a7aa1.ssl.cf2.rackcdn.com/5b78192ac9659217dbf7c9aa/screenshot.png"
+}
```

Comment:
There are three more fixtures, I'll send you those via PM on `chat.zulip.org` :)